src/tests/ftest/util/test_utils_pool.py

```diff
@@ -532,6 +532,49 @@ def check_pool_daos_space(self, s_total=None, s_free=None):
             for index, item in enumerate(val)]
         return self._check_info(checks)
 
+    def check_pool_free_space(self, expected_scm=None, expected_nvme=None,
```

Comment:
Minor - Since this belongs to the `TestPool` class now, could we simplify the function name?
```suggestion
    def check_free_space(self, expected_scm=None, expected_nvme=None,
```